client/src/data_layer_configs/population_by_census_block_config.ts

```diff
@@ -70,10 +71,20 @@ export const LEGEND_VALUES_COUNTY = [
   },
 ];
 
-const legendInfo: LegendInfo = {
-  title: 'U.S. Census data',
-  buckets: LEGEND_VALUES_STATES,
-  bucketLabelType: FeaturePropertyDataType.Number,
+const createLegends = (): Map<GeographicLevel, LegendInfo> => {
```

Comment:
why do this in a function rather than having a static list?

Reply:
I updated the other configs that didn't need a spread operator to be consts